bitrise/print.go

```diff
@@ -38,45 +38,55 @@ func isUpdateAvailable(stepInfo stepmanModels.StepInfoModel) bool {
 }
 
 func getTrimmedStepName(stepRunResult models.StepRunResultsModel) string {
-	iconBoxWidth := len("   ")
-	timeBoxWidth := len(" time (s) ")
-	titleBoxWidth := stepRunSummaryBoxWidthInChars - 4 - iconBoxWidth - timeBoxWidth - 1
+	iconBoxWidth := len("|   ")
+	timeBoxWidth := len("| time (s) |")
+	titlePaddingLeft := len("| ")
+	titlePaddingRight := len(" ")
+
+	titleLeft := ""
+	titleMiddle := ""
+	titleRight := ""
 
 	stepInfo := stepRunResult.StepInfo
 
-	title := ""
 	if stepInfo.Step.Title != nil && *stepInfo.Step.Title != "" {
-		title = *stepInfo.Step.Title
+		titleMiddle = *stepInfo.Step.Title
+	}
+
+	isGitURL := (stepInfo.Library == "git")
+	hasTitle := (titleMiddle != stepInfo.ID+"@"+stepInfo.Version)
```

Comment:
// has the step user defined title?
// stepInfo.Step.Title is always ste, if not user defined if tis a composite of (step?id@step_version)
hasCustomTitle